docs/get-started.md

```diff
@@ -74,15 +74,15 @@ This will set environment variables that the Docker client will read which
 specify the TLS settings. Note that you will need to do that every time you
 open a new tab or restart your machine.
 
-To see what will be set, run `docker-machine env dev`.
+To see what will be set, we can run `docker-machine env default`.
 
-    $ docker-machine env dev
+    $ docker-machine env default
```

Comment:
I did change these since I noticed they were inconsistent, but if you feel it's out of scope for the PR let me know.